test/integration/targets/aws_lambda/tasks/main.yml

```diff
@@ -0,0 +1,409 @@
+---
+#
+#  Author: Michael De La Rue
+#  based on ec2_key.yml + lambda.py
+
+- block:
+
+    # ============================================================
+    - name: test with no parameters
+      lambda:
+      register: result
+      ignore_errors: true
+
+    - name: assert failure when called with no parameters
+      assert:
+        that:
+           - 'result.failed'
+           - 'result.msg.startswith("missing required arguments: name")'
```

Comment:
This could be replaced by a sanity check which ensures that AnsibleModule (or AnsibleAWSModule) gets a minimum set of required arguments or even verifies that the minimum arguments for the AWS API calls always arrive.  For now I think it's the only thing checking that the module implements that kind of argument checking so I think it should stay.